### PR TITLE
Feature/insert results panel

### PR DESCRIPTION
**Pull request by MagdalenaGie**

### CODE REVIEW

File: front/src/components/InsertResultsPanel/InsertResultRow/InsertResultRow.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <li className="game game-bottom ">
+                {p2.pos}. {p2.fullname !== "" ? p2.fullname : "wolne przejście"}
+                {p2.fullname !== "" ? (
+                    <input
+                        className="InputResult"
+                        type="number"
+                        onChange={handleScoreChange2}
+                        value={score2}
+                    />
+                ) : null}
```
Comment (pieczonkam) [inline front/src/components/InsertResultsPanel/InsertResultRow/InsertResultRow.js:40]: This fragment of a code is repetitive. Could be extracted to a function.

File: front/src/components/InsertResultsPanel/InsertResultRow/InsertResultsRow.test.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    it("should have one numeric inputs", () => {
+        let numInput = screen.getAllByRole("spinbutton");
+        for (let el of numInput) {
+            expect(el).toBeInTheDocument();
+        }
+        expect(numInput).toHaveLength(1);
+    });
+
+    it("shoulr render '/wolne przejście/", () => {
```
Comment (pieczonkam) [inline front/src/components/InsertResultsPanel/InsertResultRow/InsertResultsRow.test.js:107]: Fix typo

File: front/src/components/InsertResultsPanel/InsertResultRow/InsertResultsRow.test.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        let przejscie = screen.getByText(/wolne przejście/i);
```
Comment (pieczonkam) [inline front/src/components/InsertResultsPanel/InsertResultRow/InsertResultsRow.test.js:108]: Translate variable's name to english

File: front/src/components/InsertResultsPanel/InsertResultsPanel.test.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        }
+    });
+
+    it("should have numeric inputs", () => {
+        let inputs = screen.getAllByRole("spinbutton");
+        for (let el of inputs) {
+            expect(el).toBeInTheDocument();
+        }
+    });
+});
```
Comment (pieczonkam) [inline front/src/components/InsertResultsPanel/InsertResultsPanel.test.js:22]: Consider adding some extra tests, for instance you could check how many lists there are in the document or whether competitors' names are displayed.

**Review by pieczonkam — CHANGES_REQUESTED** [review]

**Review by pieczonkam — APPROVED** [review]

**Review by frogi16 — APPROVED** [review]

**Pull request merged**